### ISSUE TITLE
Travelers generates `Character with no location` errors in `error.log`

### ISSUE THREAD

**Issue by pharaox** [pharaox/travelers#2]
Travelers generates `Character with no location` errors in `error.log`. These errors are also reported to be related to CK3 crashes by some players, see these [crash dumps](https://github.com/abc87110/CK3-Crashes/blob/main/crashes.7z).

**Comment by pharaox** [comment]
The Travelers errors in the attached crash dumps are similar to the following:

```
[14:17:48][jomini_script_system.cpp:276]: Script system error!
  Error: Character with no location in link 'location'
  Script location: file: events/travl_travel_events_01.txt line: 27

[14:17:48][jomini_script_system.cpp:276]: Script system error!
  Error: Event target link 'location' returned an unset scope
  Script location: file: events/travl_travel_events_01.txt line: 27

[14:17:48][jomini_script_system.cpp:276]: Script system error!
  Error: Invalid right side during comparison 'location'
  Script location: file: events/travl_travel_events_01.txt line: 27
```

These errors suggest that there are characters in the game without a location, that are furthermore non-pool characters. This is rather unexpected, and so I have not encountered such errors myself while testing with the vanilla game. Most likely such characters are created by one of the many mods when the crash happened. Still, such errors should be easy to avoid by adding a check if the character location exists before using it.

**Comment by pharaox** [comment]
@abc87110 I will fix the errors in the next Travelers version, but I don't think they are related to the crashes you experienced.

* Both crashes are due to different reasons, as can be seen in `exception.txt`. One is related to the video subsystem, since the function that fails is `NVAPI_Thunk (+ 79217)` (NV stands for NVidia). The other one may have a different root cause, since the failing function is in `ck3.exe`.
* The above Travelers errors are repeated thousands of times, for hours before the crash. This suggests that they are not harmful enough to cause a crash.
* When a crash is caused by a script, the very last error in error.log may indicate the failing script line. These last errors in both log files are not from Travelers.

I can't tell exactly what caused the crashes, but I would suspect an issue between CK3 and the video subsystem.

**Comment by abc87110** [comment]
Thank you for your email. I appreciate the time you took to reach out and your efforts in addressing the issues in the upcoming Travelers version.

**Comment by pharaox** [comment]
@abc87110 The issue should be fixed in version 0.3.5 with 0359f531939774ad5d5d9871b92704c0cb6ac70f, so you should no longer see such errors. Please let me know if for some reason this is not the case.
